### ISSUE TITLE
CLUBForCategorical estimating p(y|x) , why not pass logits through a softmax activation?

### ISSUE THREAD

**Issue by ZohraRezgui** [Linear95/CLUB#23]
Thank you for this very useful work !
I just realized that in the code for CLUBForCategorical, you use directly the output of the network to calculate the cross entropy with the labels. Is there a reason why the softmax activation is avoided ?

**Comment by ZohraRezgui** [comment]
oh my bad it's included in F.cross_entropy()
